sympy/combinatorics/tests/test_coset_table.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 from sympy.combinatorics.fp_groups import FpGroup
 from sympy.combinatorics.coset_table import (CosetTable,
-                    coset_enumeration_r, coset_enumeration_c)
+                    coset_enumeration_r, coset_enumeration_c, modifed_scan_and_fill, modified_coincidence, modified_define, modified_merge, modified_rep, modified_scan)
```

Comment:
This should be split into multiple lines for ease of reading from a standard-size terminal window. I think the limit is about 80 symbols per line.